generation/localizedstringkit/__init__.py

```diff
@@ -16,35 +16,44 @@
 from localizedstringkit.files import localizable_files
 
 
-SOURCE_CODE_FILE_NAME: str = "source_strings.m"
-BUNDLE_NAME: str = "LocalizedStringKit.bundle"
-
 log = logger.get()
 
 
-def generate_code_strings_file(code_files: List[str]) -> str:
-    """Generate a single code file with all strings.
+def generate_code_strings_file(code_files: List[str]) -> dict[str: str]:
```

Comment:
Amending this to return map of bundleName: code_string_file for ease of segmenting the source/strings in generation process.

Reply:
Actually, since we use that style for all the other types (such as `List`) can you change this to conform? Same below.

Reply:
Sure